configs/sst_cloud_experience.yaml

```
@@ -8,6 +8,20 @@ data:
   packages:
     - efs-utils
     - python3-botocore
+    - python3-prompt-toolkit
+    - python3-ruamel-yaml-clib
```

```suggestion
    - python3-ruamel-yaml
    - python3-ruamel-yaml-clib
```
These come from separate SRPMs and both are required.

---

Fixed.